crates/next-custom-transforms/src/transforms/server_actions.rs

```diff
@@ -1070,12 +1089,15 @@ impl<C: Comments> VisitMut for ServerActions<C> {
     }
 
     fn visit_mut_fn_decl(&mut self, f: &mut FnDecl) {
+        let old_this_status = self.this_status.clone();
+        self.this_status = ThisStatus::Allowed;
```

Comment:
i think this might be the more idiomatic way, we don't need to clone `this_status` here, just move it out (even if it's a cheap/trivial clone)
```suggestion
        let old_this_status = std::mem::replace(self.this_status, ThisStatus::Allowed);
```
(same for the exit)